.gitignore

```
@@ -1,16 +1,70 @@
-*tmp*
-/Basic_Python_Diploma/downloads/*
 access_token.json
```

Если вы не хотите, чтобы этот файл попадал в коммиты независимо от того, в каком месте он лежит, то нужно сделать вот так:

```suggestion
**/access_token.json
```

---

Ок